src/rust/multi.bato/res/source.json

```diff
@@ -0,0 +1,157 @@
+{
+	"info": {
+		"id": "multi.bato",
+		"lang": "multi",
+		"name": "BatoTo",
+		"version": 1,
+		"url": "https://bato.to",
+		"nsfw": 1
```

Comment:
should probably be `2` since there are no filters to hide it (first thing i saw when opening the source was nsfw stuff)

Reply:
updated the nsfw value to 2